presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -195,6 +195,8 @@ Property Name                                      Description
                                                    S3SelectPushdown.
 
 ``hive.metastore.load-balancing-enabled``          Enable load balancing between multiple Metastore instances
+
+``hive.orc-use-vector-filter``                     Enable use of vector ORC readers in compilation and execution ``false``
```

Comment:
```suggestion
`hive.orc-use-vector-filter``                     Enable use of vector ORC readers in compilation and          ``false``
                                                   execution
```
So tables are fussy, and leaving "execution" on line 199 causes it to break into the Default values column. When I edited hive.rst in my local build according to this screenshot, the page compiles successfully and your added property is present. 
(I think my suggestion here is correct, I'm including a screenshot of my Visual Studio Code for the suggested edit to show you exactly what worked to fix this in my local build.) 

<img width="1014" alt="Screenshot 2024-06-12 at 3 01 23 PM" src="https://github.com/prestodb/presto/assets/7013443/da458034-e59f-443a-bced-e149f5c1bb3c">

Reply:
Issue fixed
![Fix](https://github.com/prestodb/presto/assets/100258271/a04bb46f-3b1b-436e-ae30-ae6f56189bd0)